# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/obs-studio/no-glvnd.patch
# Copyright (C) 2022 - 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

cmake refuses to find EGL without glvnd, ...

CMake Error at /usr/share/cmake-3.27/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find OpenGL (missing: OPENGL_opengl_LIBRARY)
Call Stack (most recent call first):
  /usr/share/cmake-3.27/Modules/FindPackageHandleStandardArgs.cmake:600 (_FPHSA_FAILURE_MESSAGE)
  /usr/share/cmake-3.27/Modules/FindOpenGL.cmake:545 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  libobs-opengl/cmake/legacy.cmake:59 (find_package)
  cmake/Modules/ObsHelpers.cmake:519 (include)
  libobs-opengl/CMakeLists.txt:3 (legacy_check)

--- obs-studio/libobs-opengl/cmake/legacy.cmake.vanilla	2023-09-28 19:01:33.169000000 +0200
+++ obs-studio/libobs-opengl/cmake/legacy.cmake	2023-09-28 19:02:02.635000000 +0200
@@ -56,10 +56,10 @@
   set_target_properties(libobs-opengl PROPERTIES PREFIX "")
 
   if(ENABLE_WAYLAND)
-    find_package(
-      OpenGL
-      COMPONENTS EGL
-      REQUIRED)
+    find_package(PkgConfig REQUIRED)
+    pkg_check_modules(EGL REQUIRED egl IMPORTED_TARGET GLOBAL)
+    add_library(OpenGL::EGL ALIAS PkgConfig::EGL)
+
     find_package(Wayland REQUIRED)
 
     target_sources(libobs-opengl PRIVATE gl-wayland-egl.c)
